roles/kvirt_vm/tasks/create-vm.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "Validate VM name"
+  assert:
+    that: vm.name is defined
+    fail_msg: VM name is required
+
+- name: Create the VM Namespace for {{ vm.name }}
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ vm.namespace | default(kvirt_vm_namespace) }}"
+        labels:
+          security.openshift.io/scc.podSecurityLabelSync: "false"
+          pod-security.kubernetes.io/enforce: "baseline"
+          pod-security.kubernetes.io/enforce-version: "latest"
+
+- name: Delete VM {{ vm.name }}
+  ansible.builtin.include_tasks: delete-vm.yml
+  when:
+    - vm.force | default(kvirt_vm_force) | bool
```

Comment:
Ah this one should be in the main, I think we missed it during the rebase

Reply:
I moved it here because we have access to the VM name in the loop.

Reply:
You could use another loop to call the delete, same way as in the create

Reply:
Done, thanks!